core/net/mac/tsch/tsch-queue.h

```diff
@@ -39,6 +39,7 @@
 #include "lib/ringbufindex.h"
 #include "net/linkaddr.h"
 #include "net/mac/tsch/tsch-schedule.h"
+#include "net/mac/mac.h"
```

Comment:
Why was this needed in this `.h` file? (I find this odd given there are no other changes to the file)
Shouldn't this be rather included in some particular `.c` files?